src/main/java/org/polyjdbc/core/query/InsertQuery.java

```diff
@@ -100,24 +86,11 @@ String getSequenceField() {
     public InsertQuery value(String fieldName, Object value) {
         valueNames.append(fieldName).append(", ");
         values.append(":").append(fieldName).append(", ");
-        query.setArgument(fieldName, value);
+        setArgument(fieldName, value);
         return this;
     }
 
-    /**
-     * Manually set sequenced field value.
-     */
-    public InsertQuery sequenceValue(long value) {
```

Comment:
I don't think this method is used by polyjdbc clients. 
Typically you run queries via polyjdbc QueryRunner:

```
        InsertQuery query = javersPolyJDBC.query().insert().into(getSnapshotTableNameWithSchema())
                .value(SNAPSHOT_TYPE, cdoSnapshot.getType().toString())
                .value(SNAPSHOT_GLOBAL_ID_FK, globalIdPk)
                .value(SNAPSHOT_COMMIT_FK, commitIdPk)
                .value(SNAPSHOT_VERSION, cdoSnapshot.getVersion())
                .value(SNAPSHOT_STATE, jsonConverter.toJson(cdoSnapshot.getState()))
                .value(SNAPSHOT_CHANGED, jsonConverter.toJson(cdoSnapshot.getChanged()))
                .value(SNAPSHOT_MANAGED_TYPE, cdoSnapshot.getManagedType().getName())
                .sequence(SNAPSHOT_PK, getSnapshotTablePkSeqWithSchema());

        return javersPolyJDBC.queryRunner().insert(query);
```
and QueryRunner increments the sequence and saves the last value. Why to bypass the flow?